### DISCUSSION TITLE
Shortcut to set the opacity of one or several columns

### DISCUSSION THREAD

**Discussion by ghost** [opentoonz/opentoonz#6648] [General]
### Is your feature request related to a problem? Please describe

It can be time consuming to have to manually set the percentage of a column (impact head here) each time for usual clean up tasks.

![shotcuts for opacity](https://user-images.githubusercontent.com/29790512/198846475-7dc21acf-5a8f-4ba7-bb4f-3ed8871ad63c.gif)

### Describe the solution you'd like

A request to have shortcuts to be able to set the percentage of opacity of a column (s) to a certain value (which could be changed in the preferences eventually) for example to 30%.

To have the possibility to do it on all the columns or the selected one. Similar to what already exists for the other attributes of the column header.

**Comment by RodneyBaker** [comment]
Not quite the same things but... In some cases it may be useful to select a group of columns and collapse them into a subxsheet.  Then tinting and opacity can be assigned to all of those columns at the same time.

**Comment by RodneyBaker** [comment]
Transferring to Discussion as the author is a ghost.
